docs/CONAN_USER_DOCUMENTATION.md

```diff
@@ -1,3 +1,18 @@
-# Documentation
+# How to use Conan with NXRM
 
-Coming soon!
+This guide assumes that you have already installed the Conan plugin into NXRM - instructions for doing so can be found
+in the main Readme.md file at the root of the GitHub project.
+
+1. Install Conan (if on OSX, you can do this via ```brew install conan```)
+2. Create the conan repositories in NXRM (e.g. conan-hosted)
+3. Add the conan security realm to NXRM: Login as an admin, go to settings > security > realms and add 
+   ```Conan Bearer Token Realm```
+4. (Optional) Conan will already have a remote configured so if you want it to exclusively use NXRM you will need to 
+    remove that remote first via ```conan remote remove conan-center```. *Warning: If you do not do this conan will
+    fetch from conan-center first and will circumvent NXRM*
```

Comment:
NOTE: I'm wishy washy if there should be a period at the end of the warn.  It doesn't quite fall into the rules of end multiple sentences bullets with a period since it's a warn but I'd lean to it (gramatically correct).